aws-android-sdk-s3/src/main/java/com/amazonaws/mobileconnectors/s3/transferutility/TransferStatusUpdater.java

```diff
@@ -145,6 +146,13 @@ synchronized void removeTransfer(int id) {
      * @param id id of the transfer to remove
      */
     synchronized void removeTransferRecordFromDB(final int id) {
+        // Remove temporary file
+        if (dbUtil.getTransferById(id) != null) {
+            if (dbUtil.getTransferById(id).file.startsWith("aws-s3")) {
```

Comment:
Can you define "aws-s3" as a constant somewhere that can be referenced by both `TransferStatusUpdater` and `TransferUtility`?